docker-compose.yml

```diff
@@ -19,13 +19,23 @@ services:
       - "5000:5000"
     volumes:
       - images:/app/wwwroot/usercontent
+      - "./CollAction:/app"
     environment:
-      - ASPNETCORE_ENVIRONMENT=Release
+      - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=http://*:5000
+    command: [ "run" ]
+
+  collaction-frontend:
```

Comment:
Question, why do we have two entries for both the frontend and the backend? They're run in the same container right?

Reply:
There is a process watching the frontend for rebuilds and a process running the backend. We should have a container for each process.